code/lib/util.js

```diff
@@ -26,11 +25,11 @@ function findItems(items, searchTerm) {
    var matches = []
    //a lot can be improved here to improve matching the items
    for (var i=0; i<items.length; i++) {
-      if (textLib.fuzzyMatch(items[i].shirt.title, searchTerm)) {
+      if (items[i].shirt.title.concat("s").toLowerCase().includes(searchTerm.toString().toLowerCase())) {
```

Comment:
In general we shouldnt do any NL side logic in the JS. what if we add a shirt called "Penny" where the plural is "Pennies" and not "Pennys"? Your search term "Pennies" would not be included in the concat'd title, "Pennys"

Reply:
Also, you need to guard this with the english language which adds even more complexity to JS